contents/docs/getting-started/group-analytics.mdx

```diff
@@ -23,10 +24,14 @@ import JSSetGroupProperties from "../\_snippets/setting-group-properties.mdx"
 
 <JSCreateGroups />
 
-### Setting and updating group properties
+## How to set group properties
 
 <JSSetGroupProperties />
 
+## How to capture group events
```

Comment:
```suggestion
## How to capture non-user group events
```

Reply:
Idk if this is the right name

Reply:
The way I would think about this instead is there's always a person from the company who decided to do the event, or you can assume the owner of the company did some event, e.g. company_upgraded_plan <- there was someone who did that.
Having this section could be more confusing than helpful 🤷‍♀️

Reply:
so 2 cases:
- a person from a company did something, e.g. upgrade plan
- we want to set group properties, e.g. plan = premium

The event for a group won't really play nicely with group analytics - as in it will still be a person tied to the group or not in the end iiuc. What do other product analytics platforms do in these cases?